docs/source/payout.rst

```diff
@@ -0,0 +1,51 @@
+Distribute Dividends
+====================
+
+The command line tool supports a simple CSV input to make payouts to shareholders.
+
+* Shares are exchanged here for ether or tokens like Gemini USD.
```

Comment:
Shares itself are not exchanged/given (away). If you own security tokens, you are entitled to the payouts.